content/_sass/partials/_structure.scss

```diff
@@ -0,0 +1,131 @@
+// body {
+//   padding: 61px 0 0;
+// }
+
+/**
+ * Fix for Sticky Footer
+ */
+.site-wrapper {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  padding: 61px 0 0;
+}
+
+.content-wrapper {
+  // Double-check if these are equivalent
+  flex: 1 0 auto;
+  flex: 1;
+}
+
+.container-md,
+.container {
+  max-width: initial;
+}
+
+@include media-breakpoint-up(md) {
+  .container-md,
+  .container {
+    max-width: 750px;
+  }
+}
+
+// Content-Area Width
+@include media-breakpoint-up(lg) {
+  .container-lg,
+  .container {
+    max-width: 970px;
+  }
+}
+
+@include media-breakpoint-up(xl) {
+  .container-xl,
+  .container {
+    max-width: 1170px;
+  }
+}
+
+/**
+ * Header Offset Fix for Anchor Links
+ */
+h2::before {
+  content: " ";
+  display: block;
+  height: 65px;
+  margin-top: -65px;
+  pointer-events: none;
+  visibility: hidden;
+  z-index: 1;
+}
+
+h3::before {
+  content: " ";
+  display: block;
+  height: 65px;
+  margin-top: -65px;
+  pointer-events: none;
+  visibility: hidden;
+  z-index: 1;
+}
+
+.anchorfix {
+  position: relative;
+  top: -65px;
+
+  span {
+    position: relative;
+    top: 65px;
+  }
+}
+
+.longfix {
+  position: relative;
+  top: -100px;
+
+  span {
+    position: relative;
+    top: 100px;
+  }
+}
+
+@include media-breakpoint-up(md) {
+  .longfix {
+    top: -65px;
+
+    span {
+      top: 65px;
+    }
+  }
+}
+
+/**
+ * Tables
+ */
+
+.table {
+  margin: 0 0 1.25rem;
+  // width: initial;
+
+  th,
+  td {
+    padding: 0.5rem;
+    width: 30%;
+  }
+
+  thead th {
+    border-top: 0;
+  }
+
+  a {
+    word-break: break-all;
```

Comment:
I think this should be `work-break: normal` otherwise the 1st column of the tables on the main page have awkward text breaks.

Reply:
Yes - done.